taskcluster/ci/update-project/kind.yml

```diff
@@ -17,6 +17,7 @@ job-defaults:
     worker:
         docker-image: {in-tree: l10n}
         max-run-time: 3600
+    pr-target: master
```

Comment:
I think we should switch the default here, even though the most common value is currently `master`, as I expect we'll eventually switch all of them.

I'd also be inclined to leave the default implicit here, but I don't feel strongly about that. (If we have it explicit here, I would not have a default in the transform.)